### DISCUSSION TITLE
discovery.launch-options.timeout not respected

### DISCUSSION THREAD

**Discussion by miiimooo** [percy/cli#658] [Support] [answered]
## The problem

In my .percy.yml I have a discovery section that contains this:

```
discovery:
  launchOptions:
    timeout: 60000
```
When running snapshot I get these Error messages
```
[percy] Error: Navigation failed: Timeout of 30000ms exceeded.
```
This implies the timeout setting in the yaml file isn't passed on

The timeout setting is documented here https://docs.percy.io/docs/cli-configuration#a-complete-config

Also tested with the alternative spelling of `launch-options.timeout`
```
[percy] Error: Navigation failed: Timeout of 30000ms exceeded.
```

## Environment

- Node version: v12.19.0
- `@percy/cli` version: 1.0.0-beta.71
- Version of Percy SDK you’re using: 
- If needed, a build or snapshot ID:
- OS version:
- Type of shell command-line [interface]:

**Comment by Robdel12** [comment, marked as answer]
Hey @miiimooo! That config option doesn't alter the navigation timeout, that is for the browser launch timeout

>  Timeout for how long the browser will try to launch. Default: 30000

The navigation is not configurable, if it's taking longer than 30 seconds to complete navigation to a page there's an issue. It'd be worth looking at debug logs (`--debug`) to figure out what might be hanging the navigation: https://docs.percy.io/docs/debugging-sdks#debugging-sdks Also might be worth observing what the asset discovery browser is doing: https://docs.percy.io/docs/debugging-sdks#display-the-asset-discovery-browser
